be/src/olap/memtable.h

```diff
@@ -195,9 +199,11 @@ class MemTable {
     //for vectorized
     vectorized::MutableBlock _input_mutable_block;
     vectorized::MutableBlock _output_mutable_block;
-    vectorized::Block _collect_vskiplist_results();
+    void _collect_vskiplist_to_output(bool final);
```

Comment:
nit: cpp has `final` keyword, it's better to use another name,  maybe `is_final`?